src/landing/LoginForm.js

```diff
@@ -88,10 +97,12 @@ const LoginForm = ({ onSubmit }) => {
 
 LoginForm.defaultProps = {
   onSubmit: () => {},
+  loginStatus: null,
 };
 
 LoginForm.propTypes = {
   onSubmit: PropTypes.func,
+  loginStatus: string,
```

Comment:
Code should be consistent, either both use PropTypes:
```
onSubmit: PropTypes.func,
loginStatus: PropTypes.string,
```
or both use without PropTypes: 
  
```
onSubmit: func,
loginStatus: string,
```

Reply:
Updated